template/configure.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 . .git/hooks/hook_switcher.sh
 
+if [[ "$OSTYPE" == *darwin* ]]
+then
+  READLINK_CMD='greadlink'
+else
+  READLINK_CMD='readlink'
+fi
+ 
 readonly PROGNAME=$(basename "$0")
-readonly PROGDIR=$(readlink -m "$(dirname "$0")")
+readonly PROGDIR=$($READLINK_CMD -m "$(dirname "$0")")
```

Comment:
`cd -` is a shell feature, and it is unlikely but can not work.
It will be safer if you use the option with `dirname`